### DISCUSSION TITLE
Broken links on vanderbilt.edu

### DISCUSSION THREAD

**Discussion by mitza-oci** [DOCGroup/ACE_TAO#2137] [General]
There are some links to documentation like this https://www.dre.vanderbilt.edu/~schmidt/DOC_ROOT/ACE/docs/ACE-porting.html that don't seem to be working anymore.  Does anyone know what changed?  If there's a new URL for this then some existing pages on vanderbilt.edu (and in OpenDDS's docs) need to change too.

**Comment by mitza-oci** [comment]
See https://www.dre.vanderbilt.edu/~schmidt/ACE-documentation.html for an example of a page that has broken links

**Comment by jwillemsen** [comment]
@douglascraigschmidt Do you know what is happening with your website, have you removed your local copy of ACE/TAO?

**Comment by jwillemsen** [comment]
Homepage is working again
